worker/expiration_worker.go

```diff
@@ -238,22 +233,27 @@ func (w *ExpirationWorker) Run() {
 			}
 			if w.running == false {
 				w.running = true
-				result, err := w.expireScores()
+				results, err := w.expireScores()
 				if err != nil {
-					l.Error("error expiring scores", zap.Error(err))
+					errChan <- fmt.Errorf("error expiring scores: %v", err)
+					break
```

Comment:
Are you sure we should stop the worker if any error occurs? I think we should exit if we reach some threshold (e.g. if redis is down) but not on the first error.

Reply:
Reading about redis internally it seems that they keep trying to execute operations specifically when there are network errors, eventually sleeping if there are enough errors. Restarting processing if the server goes back online. So I think the best way to solve this, for now, is to use an error channel merely for reporting and keep the worker running.

Reply:
@cscatolini PTAL